analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -596,6 +596,16 @@ class Pip(
                 }
             }
 
+            val projectUrls = pkgInfo["project_urls"]
+            val vcsFallbackUrls = listOfNotNull(
+                homepageUrl,
+                pkgInfo["project_url"]?.textValue(),
+                projectUrls["Code"]?.textValue(),
+                projectUrls["Homepage"]?.textValue(),
```

Comment:
Nit: Does it make sense to swap the order here, to move `"Homepage"` closer to `homepageUrl` and `"Code"` closer to `"Source Code"`?

Reply:
I'd rather keep this list alphabetically sorted.